ha/core/controllers/pcs/node_controller.py

```diff
@@ -114,7 +144,24 @@ def start(self, nodeid: str) -> dict:
             ([dict]): Return dictionary. {"status": "", "msg":""}
                 status: Succeeded, Failed, InProgress
         """
-        raise HAUnimplemented("This operation is not implemented.")
+        _res = self.status([nodeid])
```

Comment:
for output check docstring
Output should be like 

```
{"status": "", "msg":""}
```

Here status is InProgress and msg is some text success or something